src/host-web/chucknode-postjs.js

```diff
@@ -509,6 +509,24 @@ class ChuckNode extends AudioWorkletProcessor
                 {
                     return function()
                     {
+                        // Create directories required to load filesToPreload
+                        const dirs = filesToPreload.map(
+                            (f) => {
+                                const path = f.filename.split("/");
+                                return path.slice(0, path.length - 1);
+                            }
+                        );
+
+                        dirs.forEach(
+                            (dir) =>
+                                dir.reduce(
+                                    (parent, name) => {
+                                        Module["FS_createPath"](parent, name, true, true);
+                                        return (parent + "/" + name);
+                                    },
+                                    "/"
+                                );
+                        );
                         // Create a directory for chugins
                         Module["FS_createPath"]("/", "chugins", true, true);
```

Comment:
I tested this and it looks like this line becomes redundant! Thanks for making it flexible to handle this!